### PR TITLE
Printer config fixes

### LINKED ISSUE #68: Printer configs are wrong / links broken
As per user @_blkmrkt on Discord:

## Z and E swapped in the config?

I think that extruder and stepper_z are swapped in this template:
https://docs.vzbot.org/assets/images/manual/vz330_mellow/electronics/Printer_config/printer.cfg

That config doesn't agree with this diagram: https://raw.githubusercontent.com/VzBoT3D/VzBoT-Vz330/master/Wiring%20Diagram/VZBOT%20330%20WIRING%20DIAGRAM%20AWD%20WITH%20CPAP.png

---

## Broken hyperlink

This hyperlink is broken on https://docs.vzbot.org/vz330_mellow/electronics/Printer_Config btw, I had to manually replace the %-encoded symbols in order to download it.
It looks like this currently: https://docs.vzbot.org/assets%5Cimages%5Cmanual%5Cvz330_mellow%5Celectronics%5CPrinter_config%5Cprinter.cfg

---

## CPAP pin low on startup

I'd also add a line about setting the CPAP pin to LOW when configuring the STM32 firmware, otherwise it's a menace. Adding !PF8 to GPIO pins to set at microcontroller startup helped in my case.

Perhaps you could insert this screenshot/step into https://docs.vzbot.org/vz330_mellow/electronics/Firmware? It'll prevent hearing loss for billions of people. `https://github.com/VzBoT3D/docs/assets/16231288/96002d76-4bcc-410e-b818-6e0b0226048f`

### LINKED ISSUE #86: printer.cfg replacement in the docs?
Hello - can the file here on Github named 330AWD-printer.cfg replace the printer.cfg linked at https://docs.vzbot.org/vz330_mellow/electronics/Printer_Config - the link there seems to be broken? Thanks

### PR DESCRIPTION
**Pull request by HumbleDeer**
Fixes #68 ("Printer configs are wrong / links broken")
Incorporates #75 ("updated the the link to 330AWD-printer.cfg") ⚠️Currently broken

### CODE REVIEW

**Comment by HumbleDeer** [comment]
Checked for other broken links, and all seems fine for now.

**Comment by HumbleDeer** [comment]
Added commits from https://github.com/VzBoT3D/docs/tree/pbsuper-patch-1 to this branch and PR

File: docs/assets/images/manual/vz330_mellow/electronics/Printer_config/printer.cfg [hunk 1]
```diff
@@ -212,10 +212,10 @@ heater_pin: PB0
 sensor_type: PT1000
 sensor_pin: PF4
 
-#control: pid
-#pid_Kp=28.737 
-#pid_Ki=1.935 
-#pid_Kd=106.684
+control: pid
```
Comment (HumbleDeer) [inline docs/assets/images/manual/vz330_mellow/electronics/Printer_config/printer.cfg:214]: These values use `pid_xx: value` while the ones below use `pid_xx=value`. I assume this is not correct?

**Comment by HumbleDeer** [comment]
Test deployment succeeds

Ready to merge. I'll do the other fixes in another PR?

**Review by pbsuper — CHANGES_REQUESTED** [review]
Printer.cfg doesn't match wiring diagrams.

**Comment by HumbleDeer** [comment]
I... edited the wiring diagram to match the configs.

**Review by pbsuper — APPROVED** [review]
good.

**Pull request merged**